### DISCUSSION TITLE
Hardware transcoding PLEX/Jellyfin on Pentium Silver N6005

### DISCUSSION THREAD

**Discussion by guzior-lps** [alexbelgium/hassio-addons#860] [General]
Hi.
Is somebody successfully running hardware transcoding with one of these addons?

In plex container with command:
cd /dev/dri

i have renderD128 but when rying transcoding no (hw) sign apearing in control panel of plex. 

Can somebody explain/write me full sequence of what to do to enable HW transcoding?

**Comment by alexbelgium** [comment]
Hi, you can use this guide https://github.com/alexbelgium/hassio-addons/wiki/Add-ons-feature-:-customisation to add features to the addons, such as installing graphic drivers

renderD128 is available but there is a high probability that relevant drivers are not installed

**Comment by guzior-lps** [comment]
Main problem of mine is that i have no idea how to install it (no difference if were talking about jellyfin or about plex) and there are no many informations about that in the internet...
Maybe there is somebody that can write me in points what to do?
